spec/requests/chat_channel_memberships_spec.rb

```diff
@@ -20,8 +20,8 @@
       end
 
       it "return all details of chat channel" do
-        expect(response).to have_http_status(:ok)
-        expect(response.parsed_body["result"].keys).to eq(%w[chat_channel memberships current_membership])
+        expect(response.status).to eq(200)
+        expect(JSON.parse(response.body)["result"].keys).to eq(%w[chat_channel memberships current_membership invitation_link])
```

Comment:
Why did you remove `response.parsed_body` here? I think we should keep it since its a little cleaner and we dont have to do the parsing.